app/vmalert/notifier/template_func.go

```diff
@@ -167,6 +167,10 @@ func InitTemplateFunc(externalURL *url.URL) {
 		"queryEscape": func(q string) string {
 			return url.QueryEscape(q)
 		},
+		"crlfEscape": func(q string) string {
```

Comment:
Can we have a test for this?
I'm also thinking whether we need these 3 different escape functions? Should we have just queryEscape or pathEscape to do all the job? I mean, should the user be aware of all these functions or just have one but for all cases?

Reply:
yep, will add test.

   I think, that we can provide flexibility. For grafana introduce one function - `grafanaExpressionFormat` and use it as help suggestion, for other systems left queryEcsape and other exist functions.

Reply:
Agreed with @f41gh7 . Let's add a single function for expression escaping in Grafana, while leaving all the existing functions for covering various cases.